Integrations/BitDam/BitDam.py

```diff
@@ -0,0 +1,234 @@
+import demistomock as demisto
+from CommonServerPython import *
+'''IMPORTS'''
+import requests
+import base64
+import json
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+'''INTEGRATION PARAMS'''
+API_TOKEN = demisto.params().get('apitoken')
+URL_BASE = demisto.params().get('url')
+USE_PROXY = demisto.params().get('proxy', False)
+UNSECURE = demisto.params().get('insecure', False)
+
+'''CONSTANTS'''
+READ_BINARY_MODE = 'rb'
+SLASH = '/'
+SCAN_FILE_URL = 'direct/scan/file/'
+GET_FILE_VERDICT_URL = 'direct/verdict/?hash={}'
+TOKEN_PREFIX = 'Bearer'  # guardrails-disable-line
+RESPONSE_CODE_OK = 200
+STATUS_IN_PROGRESS = 'IN_PROGRESS'
+STATUS_DONE = 'DONE'
+
+VERDICT_SCANNING = 'Scanning'
+VERDICT_MALICIOUS = 'Malicious'
+VERDICT_APPROVED = 'Approved'
+VERDICT_ERROR = 'Error'
+VERDICT_BENIGN = 'Benign'
+VERDICT_TIMEOUT = 'Timeout'
+
+BITDAM_COMMAND_PREFIX = 'bitdam'
+DBOTSCORE_UNKNOWN = 0
+DBOTSCORE_CLEAN = 1
+DBOTSCORE_MALICIOUS = 3
+
+'''HANDLE PROXY'''
+if not USE_PROXY:
+    del os.environ["HTTP_PROXY"]
+    del os.environ["HTTPS_PROXY"]
+    del os.environ["http_proxy"]
+    del os.environ["https_proxy"]
+
+
+'''HELPER FUNCTIONS'''
+
+
+def get_file_bytes(entry_id):
+    get_file_path_res = demisto.getFilePath(entry_id)
+    file_path = get_file_path_res["path"]
+    with open(file_path, READ_BINARY_MODE) as fopen:
+        bytes = fopen.read()
+    return base64.b64encode(bytes)
+
+
+def get_url_base_with_trailing_slash():
+    '''
+    Returns the intergation's base url parameter, making sure it contains an trailing slash
+    '''
+    url_base = URL_BASE
+    return url_base if url_base.endswith(SLASH) else url_base + SLASH
+
+
+def get_auth_headers():
```

Comment:
This func is redundant, it always return the same data, should just set it in advance instead of calling it on every API call